R/oauth-flow-auth-code.R

```diff
@@ -139,12 +149,23 @@ oauth_flow_auth_code <- function(client,
                                  pkce = TRUE,
                                  auth_params = list(),
                                  token_params = list(),
-                                 host_name = "localhost",
+                                 host_name = NULL,
                                  host_ip = "127.0.0.1",
-                                 port = httpuv::randomPort()
+                                 port = httpuv::randomPort(),
+                                 redirect_url = "http://localhost"
 ) {
   oauth_flow_check("authorization code", client, interactive = TRUE)
-  check_installed("httpuv")
+
+  # For backwards compatibility, fall back to the original redirect URL
+  # construction.
+  if (!is.null(host_name)) {
```

Comment:
Can you formally deprecate `host_name`? (https://lifecycle.r-lib.org/articles/communicate.html#deprecate-an-argument-keeping-the-existing-default)

Reply:
Done.